### DISCUSSION TITLE
`involving` search query

### DISCUSSION THREAD

**Discussion by kospl** [community/community#5394] [Discussions] [answered]
![image](https://user-images.githubusercontent.com/26386660/131708717-1d26100a-3cf6-40d0-8910-d5cd6c4df8bb.png)

It seems like filter query with keyword `involving` doesn't works properly.

I was trying to search anything involving me, and expect nothing found, but there are two unrelated discussions in search results.

**Comment by laymonage** [comment, marked as answer]
I'm not sure where you got the `involving` and `user:me` queries from, but the correct query is `involves`, e.g.:

https://github.com/github/feedback/discussions/categories/discussions-feedback?discussions_q=category%3A%22Discussions+Feedback%22+involves%3Akospl

https://github.com/github/feedback/discussions/categories/discussions-feedback?discussions_q=category%3A%22Discussions+Feedback%22+involves%3Alaymonage

It is documented here: https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-discussions#search-by-a-user-thats-involved-in-a-discussion

**Comment by kospl** [comment]
My bad, English in not my native language, I think I just got it from my head `involving:@me`.
Indeed, correct query is `involves:@me`.
